public/app/core/navigation/GrafanaRouteLoading.tsx

```diff
@@ -0,0 +1,24 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { LoadingPlaceholder, useStyles2 } from '@grafana/ui';
+
+export function GrafanaRouteLoading() {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.loadingPage}>
+      <LoadingPlaceholder text={'Loading...'} />
+    </div>
+  );
+}
+
+const getStyles = () => ({
+  loadingPage: css({
+    height: '100%',
+    flexDrection: 'column',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  }),
```

Comment:
Nit/question: Any benefit of using `css({...})` rather than templated srting?

Reply:
@dprokop speed (no need to parse it), better autocomplete / IntelliSense, typechecking , easier to write and compose, I use it for all new styles